### PR TITLE
favicon

### PR DESCRIPTION
**Pull request by BeanGalSolina**

### CODE REVIEW

**Review by PoolloverNathan — APPROVED** [review]
svgs look like legal svg code

File: js/master.js [hunk 1]
```diff
@@ -23,7 +24,13 @@ function getClosestFactors(num) {
     }
 }
 
-window.onload = function () {
+window.onload = function() {
```
Comment (PoolloverNathan) [inline js/master.js:27]: this is deprecated btw

File: js/master.js [hunk 2]
```diff
@@ -23,7 +24,13 @@ function getClosestFactors(num) {
...
+    if (navigator.userAgent.includes('Firefox')) {
+        document.getElementById('favicon').setAttribute('href', 'chrome://branding/content/icon32.png')
+    }else {
+        document.getElementById('favicon').setAttribute('href', '/favicon.svg')
+    }
```
Comment (PoolloverNathan) [inline js/master.js:32]: this could be simplified to a `?:`, but I'll allow it

**Review by PoolloverNathan — APPROVED** [review]

File: js/master.js (hunk 1 above)
Comment (BeanGalSolina) [reply js/master.js:27]: shit I didn't mean to do that

File: js/master.js (hunk 2 above)
Comment (BeanGalSolina) [reply js/master.js:32]: readability

**Pull request merged**